docs/src/main/sphinx/security/built-in-system-access-control.md

```diff
@@ -13,10 +13,29 @@ cluster nodes:
 access-control.name=allow-all
 ```
 
-Multiple system access control implementations may be configured at once
-using the `access-control.config-files` configuration property. It should
-contain a comma separated list of the access control property files to use
-(rather than the default `etc/access-control.properties`).
+(multiple-access-control)=
+## Multiple access control systems
+
+Multiple system access control implementations may be configured at once using
+the `access-control.config-files` configuration property. It must contain a
+comma-separated list of the access control property files to use, rather than
+the default `etc/access-control.properties`. Relative paths from the Trino
+`INSTALL_PATH` or absolute paths are supported.
+
+The configured access control systems are used in order until access rights are
```

Comment:
I am not sure about this .. also would be good to explain more with an example.

Reply:
well.. if they are independent.. what does that mean .. are the evaluted in order . what if the first on does a deny .. but the second one would be granting access ..

Reply:
Any individual deny results in deny as the final result. So it doesn't really matter which does this first :)